lib/utils/template_generator.js

```diff
@@ -22,7 +22,11 @@ class TemplateGenerator {
     console.log(__('Installing Template from ' + uri + '....').green);
 
     fs.mkdirpSync(utils.dirname(tmpFilePath));
-    utils.downloadFile(url, tmpFilePath, () => {
+    utils.downloadFile(url, tmpFilePath, (err) => {
+      if (err) {
+        console.error(err.red);
```

Comment:
Maybe we could add a bit of flavor to it like: "Download failed. Does the templates really exist? He are embark's supported templates: https://embark.status.im/templates/" and after that the error we got

Reply:
Done.